### DISCUSSION TITLE
How to add registration field of type text into the notification email

### DISCUSSION THREAD

**Discussion by krischanskide** [derhansen/sf_event_mgt#1018] [General]
I would like to add the text of a registration field of type text into the notification email, but these are not part of {registration.fieldValues}. Is there a way to access them?

**Comment by krischanskide** [comment]
I found them in {registration.event.registrationFields}.

**Reply by derhansen** [reply]
Yes, that is correct. The content of `{registration.fieldValues}` will only contain values of fields the user actually filled out during registration.
